esrally/actor.py

```diff
@@ -224,7 +224,7 @@ def bootstrap_actor_system(try_join=False, prefer_local_only=False, local_ip=Non
                 coordinator_ip = None
                 local_ip = None
         else:
-            if system_base != "multiprocTCPBase" and system_base != "multiprocUDPBase":
+            if system_base not in ('multiprocTCPBase', 'multiprocUDPBase'):
```

Comment:
Can we please keep the double quotes?

Reply:
Oops, I was just taking the Pylint suggestion as-is.  Changing back to double quotes.